src/raw.imageio/rawinput.cpp

```diff
@@ -433,8 +433,15 @@ RawInput::open_raw(bool unpack, const std::string& name,
         m_unpacked = true;
     }
 
-    //Store flip before it is potentially overwritten
-    int original_flip = m_processor->imgdata.sizes.flip;
+    // Store flip before it is potentially overwritten
+    // LibRaw's convention for flip values differs from Exif orientation tags
+    // so we need to convert it
+    int original_flip = 1;
+    int libraw_flip = m_processor->imgdata.sizes.flip;
+    int libraw_to_exif_flip[8] = {1, 2, 4, 3, 5, 8, 6, 7};
```

Comment:
I think this change will make clang-format happy, and also let's make that array static:

```suggestion
    int original_flip                 = 1;
    int libraw_flip                   = m_processor->imgdata.sizes.flip;
    static int libraw_to_exif_flip[8] = {1, 2, 4, 3, 5, 8, 6, 7};
```